pkg/parser/parser.go

```diff
@@ -24,26 +22,17 @@ func ParseFromGit(path string, opts *git.CloneOptions) ([]*Resource, error) {
 }
 
 func parseConfig(path string, files fs.FileSystem) ([]*Resource, error) {
-	k8sfactory := k8sdeps.NewFactory()
-	ldr, err := loader.NewLoader(path, files)
-	if err != nil {
-		return nil, err
-	}
-	defer func() {
-		err = ldr.Cleanup()
-		if err != nil {
-			panic(err)
-		}
-	}()
-	kt, err := target.NewKustTarget(ldr, k8sfactory.ResmapF, k8sfactory.TransformerF)
-	if err != nil {
-		return nil, err
-	}
-	r, err := kt.MakeCustomizedResMap()
```

Comment:
Latest kustomize provides a wrapper `krusty` which has simplified functions to run kustomize build and return resources. 

ref:
- https://github.com/kubernetes-sigs/kustomize/blob/release-kustomize-v5.5.0/ARCHITECTURE.md#the-run-function
- https://github.com/kubernetes-sigs/kustomize/blob/bb7a28070905adae77c6f82b912a862de2b3a052/api/krusty/kustomizer.go#L45-L56